tests/test_enterprise/api/test_default_enrollment_views.py

```diff
@@ -0,0 +1,726 @@
+"""
+Tests for the edx-enterprise ``api.v1.views.default_enterprise_enrollments module``.
+"""
+import uuid
+from unittest import mock
+
+import ddt
+from faker import Faker
+from oauth2_provider.models import get_application_model
+from pytest import mark
+from rest_framework import status
+from rest_framework.reverse import reverse
+
+from django.conf import settings
+
+from enterprise.constants import ENTERPRISE_LEARNER_ROLE
+from enterprise.models import EnterpriseCourseEnrollment
+from test_utils import FAKE_UUIDS, TEST_PASSWORD, factories, fake_catalog_api
+
+from .constants import AUDIT_COURSE_MODE, VERIFIED_COURSE_MODE
+from .test_views import BaseTestEnterpriseAPIViews, create_mock_default_enterprise_enrollment_intention
```

Comment:
nit: should `create_mock_default_enterprise_enrollment_intention` be moved into this file, if it's only related to default enrollment views?

Reply:
I tried to do that, but the test for default realizations still lives in `test_views.py` under the bulk enrollment test class, and it needs this function, so I figured it makes more sense to keep it in the "base" test file and import it in the more specific test file.

Reply:
Ah, got it. Missed that it was still used for default realizations!